test/unit/template_engine_tests.rb

```diff
@@ -70,11 +70,18 @@ class NullTemplateEngineTests < Assert::Context
     end
 
     should "read and return the given path in its source path on `render" do
-      exists_file = 'test/support/template.json'
+      exists_file = 'test/support/template.erb'
       exp = File.read(subject.source_path.join(exists_file).to_s)
       assert_equal exp, subject.render(exists_file, @service_handler, @locals)
     end
 
+    should "complain if given a path that matches multiple files" do
+      no_exists_file = 'test/support/conflict_template'
```

Comment:
@jcredding maybe update this to do a choice on full file name or partial file name.  Then use glob* to get the path to read from.  We should do this to test that it works on both full and partial file names.